.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -11,16 +11,19 @@ an issue where no sections have been filled will be deleted without comment.
 -->
 
 **Is your feature request related to a particular plugin or theme? If so, specify it.**
-The name of the plugin or theme that you would like us to improve. [...]
+The name of the plugin, theme or alias that you would like us to improve. [...]
 
 **Is your feature request related to a problem? Please describe.**
 A description of what the problem is. Ex. I'm always frustrated when [...]
 
 **Describe the solution you'd like**
-A description of what you want to happen.
+A description of what you want to happen. [...]
 
 **Describe alternatives you've considered**
-A description of any alternative solutions or features you've considered. This can also include other plugins or aliases.
+A description of any alternative solutions or features you've considered. This can also include other plugins, themes and aliases.
```

Comment:
This should read `plugins or themes`, I don't remember when `aliases` got in here 😅

Reply:
Fixed [here](https://github.com/ohmyzsh/ohmyzsh/pull/9068/commits/f626ed522efe43f27915dafcc344341e15eeb998)